modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/taglib/servlet/taglib/ObjectDefinitionsValidationsScreeNavigationCategory.java

```diff
@@ -73,7 +75,10 @@ public String getLabel(Locale locale) {
 
 	@Override
 	public boolean isVisible(User user, ObjectDefinition objectDefinition) {
-		if (objectDefinition.isDefaultStorageType()) {
+		if ((GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-151671")) ||
+			 !objectDefinition.isSystem()) &&
+			objectDefinition.isDefaultStorageType()) {
+
```

Comment:
Hi @seltong,

I believe we can invert this conditionals, I mean something like this:

````
if (!objectDefinition.isDefaultStorageType()) {
  return false;
}

if (!objectDefinition.isSystem()) {
   return true;
}

return GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-151671");
````